pod/perlre.pod

```diff
@@ -1167,18 +1161,88 @@ X<\g{1}> X<\g{-1}> X<\g{name}> X<relative backreference> X<named backreference>
 X<named capture buffer> X<regular expression, named capture buffer>
 X<named capture group> X<regular expression, named capture group>
 X<%+> X<$+{name}> X<< \k<name> >>
-There is no limit to the number of captured substrings that you may use.
-Groups are numbered with the leftmost open parenthesis being number 1, I<etc>.  If
-a group did not match, the associated backreference won't match either. (This
-can happen if the group is optional, or in a different branch of an
-alternation.)
-You can omit the C<"g">, and write C<"\1">, I<etc>, but there are some issues with
-this form, described below.
-
-You can also refer to capture groups relatively, by using a negative number, so
-that C<\g-1> and C<\g{-1}> both refer to the immediately preceding capture
-group, and C<\g-2> and C<\g{-2}> both refer to the group before it.  For
-example:
+
+The grouping construct C<( ... )> creates a capture group (also
+referred to as a "capture buffer" or sometimes simply "capture" or
+"buffer"). There is no limit to the number of capture groups that you
+may use. Capture groups are numbered with the leftmost open parenthesis
+being number 1, the second from the left being 2, I<etc>, with one
+exception, capture groups inside of the "branch reset" construct
+C<(?| ... )> are numbered differently. (See the section on
+L<branch reset|/Branch Reset> later in this file.)
+
+Capture bufers may also be given a name by appending C<< ?<name> >>
+immediately after the open paren, for example C<< (?<blah> ... ) >>
+creates a capture group called C<blah>. This will be covered in more
+detail a bit later on in this section.
+
+After a successful match the contents of a capture group may be accessed
+via a variety of different L<special variables|perlvar/Regex Variables>
+which can be used to access the content of the buffers by number or
+name, as well as to extract the offsets into the string that was
+matched, there is also a special variable that allows access to the
+capture groups as a whole in a single array or the named captures via a
+hash.
+
+The oldest, most common and traditional way to access the contents of a
+capture buffer after a successful match is via a
+L<digit var|perlvar/Digit Vars> such as C<$1> or C<$2> I<etc>, which refer
+to the first and second capture groups in the pattern. You may refer to
+a named capture group by using C<$+{name}>. Multiple capture groups may
```

Comment:
I don't think named capture groups need to be brought up in this part, as they're discussed later - the reader hasn't been told how to declare them yet so doesn't need the explanation how to access them yet. So the remaining part of this paragraph could be added to where those are introduced.

Reply:
Actually i introduce them at line 1174.